### PR TITLE
Fix Admin not working

### PR DESCRIPTION
**Pull request by brunvez**
## Issue

Admin was not working. Now it works. Because it's fixed.

## Reason

Pundit was missing ActiveAdmin configuration

### CODE REVIEW

**Comment by brunvez** [comment]
> Do you intended to add `binding_of_caller` or is a leftover from your dark sorcery? If so, could you point me where is needed exactly?

Oh, yeah. That's a gem so that BetterError works better. It gives you a console right on the browser with the current execution context

**Comment by brunvez** [comment]
@fedeagripa here's how it looks

image

File: app/policies/application_policy.rb [hunk 1]
```diff
@@ -7,31 +7,31 @@ def initialize(user, record)
   end
 
   def index?
-    false
+    user.is_a?(AdminUser)
```
Comment (santib) [inline app/policies/application_policy.rb:10]: maybe we don't need these changes in `app/policies/application_policy.rb`

I'd have `app/policies/admin/application_policy.rb` define the policies for `AdminUser`

thoughts?

Comment (santib) [reply app/policies/application_policy.rb:10]: I say it because `User` and `AdminUser` are different models, so I think policies should be different as well and would be easier to manage

File: Gemfile [hunk 2]
```diff
@@ -43,6 +43,7 @@ end
 group :development do
   gem 'annotate', '~> 3.0', '>= 3.0.3'
   gem 'better_errors', '~> 2.5', '>= 2.5.1'
+  gem 'binding_of_caller', '~> 0.8.0'
```
Comment (santib) [inline Gemfile:46]: why is this needed?

File: app/policies/application_policy.rb (hunk 1 above)
Comment (brunvez) [reply app/policies/application_policy.rb:10]: That'd be an improvement but Pundit reflects on policies using the model, so it uses the same policies as we normally use. To change this we would need to implement some way of getting policies from the `Admin` namespace if it's for Active Admin.

I completely agree with the comment but I'd add it as an enhancement issue.

File: Gemfile (hunk 2 above)
Comment (brunvez) [reply Gemfile:46]: It's just so we can use the browser console described here https://github.com/rootstrap/rails_api_base/pull/211#issuecomment-634264205

File: app/policies/application_policy.rb (hunk 1 above)
Comment (brunvez) [reply app/policies/application_policy.rb:10]: @santib I created the issue to deal with that, but I think this should be merged ASAP https://github.com/rootstrap/rails_api_base/issues/212

Comment (santib) [reply app/policies/application_policy.rb:10]: I agree 💯
